docs: added table of contents and introduction document for Studio tool (#553)
asyncapi/studio: docs/introduction.md

Studio is a tool to develop AsyncAPI documentation. It allows you to visualize the flow, events, publishers, subscribers as you design your API.

> 

Reviewer: Maybe add more context from studio GH docs?

'One place that allows you to develop an AsyncAPI document, validate it, convert it to the latest version, preview the documentation and visualize the events flow.'

# Studio Use Cases

- Create and visualize event-driven AsyncAPI on the go.
- Realtime visualization and documentation for your APIs.

# Studio Advantages 
- User-friendly.
- Straightforward and effortless in building APIs.
- Interactive AsyncAPI documentation.

> 

Reviewer: add bullet point for 'visualizing events flow'

> # Process

Reviewer: ```suggestion
# Usage
```

> To start using Studio, [create an AsyncAPI file](https://www.asyncapi.com/docs/tutorials/streetlights#creating-the-asyncapi-file) for your API.

Reviewer: ```suggestion
To start using Studio, [create an AsyncAPI file](https://www.asyncapi.com/docs/tutorials/create-asyncapi-document#create-AsyncAPI-document) for your API. 
```

This link won't work when we merge the new 4 GSoD tutorials to replace the Streetlights tutorial: https://github.com/asyncapi/website/pull/1023/files

> To start using Studio, [create the AsyncAPI](https://www.asyncapi.com/docs/tutorials/streetlights#creating-the-asyncapi-file) file for your API.

Reviewer: ```suggestion
To start using Studio, [create an AsyncAPI file](https://www.asyncapi.com/docs/tutorials/streetlights#creating-the-asyncapi-file) for your API.
```

> 

Reviewer: ```suggestion
<Remember>

If you do not have an `asyncapi.yaml` file ready, then generate one by running: 
`asyncapi new --example=tutorial.yml --no-tty`.

</Remember>
```

> Studio is a tool to develop AsyncAPI documentation. 

Reviewer 2: We could improve that description, It's not just about AsyncAPI documentation.

A `Studio` refer to a development environment / platform, or a collection of tools that enable a certain capability.
IMO I'd define studio as an "AsyncAPI development platform (or environment) that enable users to create, visualise, collaborate on event-driven architectures"

@alequetzalli or @fmvilas might have a better description

> Using Studio you can develop an AsyncAPI document, visualize the flow of events, preview, validate, and convert the document to the latest version in one place.

Reviewer 2: I'll change `develop` here by `create`, as we don't really develop a document.
`convert the document` -> `convert the AsyncAPI document`

Reviewer: Not sure why Jagruti didn't make this suggested change, but I will do ahead and add it. :)

> Using Studio you can develop an AsyncAPI document, visualize the flow of events, preview, validate, and convert the document to the latest version in one place.

Reviewer: ```suggestion
Using Studio you can create an AsyncAPI document, visualize the flow of events, preview, validate, and convert the AsyncAPI document to the latest version in one place.
```

You can use it to view the events, publishers, subscribers, messages, payload as you design your API.

# Studio Use Cases

- Create and visualize event-driven AsyncAPI on the go.

> - Realtime visualization and documentation for your APIs.

Reviewer 2: `Realtime` -> `Real-time`
